static/scss/partials/main.scss

```diff
@@ -2429,15 +2438,4 @@ ul.messages {
     fill: $color-white !important; 
   }
 
-}
-
-.home-messages span {
-  color: $color-white;
-  text-align: center;
-  width: 100%;
-  font-weight: 600;
-}
-
-.home-messages path {
-  fill: $color-white;
-}
```

Comment:
Removed `.home-messages` and placed it into the `.messages` class instead, I did not see a need for an extra class.